README.md

```diff
@@ -1,7 +1,6 @@
-[![Build Status](https://travis-ci.org/xbmc/xbmc.svg?branch=master)](https://travis-ci.org/xbmc/xbmc)
 [![Documentation](https://codedocs.xyz/xbmc/xbmc.svg)](https://codedocs.xyz/xbmc/xbmc/)
 
-![Kodi logo](https://raw.githubusercontent.com/xbmc/xbmc-forum/master/xbmc/images/logo-sbs-black.png)
+![Kodi logo](https://raw.githubusercontent.com/xbmc/xbmc/master/media/banner.png)
```

Comment:
this can be made a relative URL I think:

```
![Kodi logo](media/banner.png)
```

Reply:
done